arango/http.py

```diff
@@ -128,7 +205,7 @@ def send_request(
             data=data,
             headers=headers,
             auth=auth,
-            timeout=self.REQUEST_TIMEOUT,
+            timeout=self.request_timeout,
```

Comment:
the result of this is that `pool_timeout` which maps to `timeout` on the connection pool, will always be ignored https://urllib3.readthedocs.io/en/stable/reference/urllib3.connectionpool.html#urllib3.HTTPConnectionPool.urlopen